src/test/kotlin/org/rust/lang/core/resolve/RsResolveTest.kt

```diff
@@ -591,5 +591,73 @@ class RsResolveTest : RsResolveTestBase() {
                 //^ unresolved
         }
     """)
-}
 
+    fun testMethod() = checkByCode("""
```

Comment:
And I believe there should be tests already for inherent and non-inherent methods, so only the one with conflict should be added.